src/MetaMorpho.sol

```diff
@@ -355,6 +356,7 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
                 }
 
                 delete config[id];
+                delete pendingCap[id];
```

Comment:
shouldn't we revert instead of removing the cap ?

Reply:
This would be consistent with #353 indeed. I would wait for it to be finalized and merged though